website_search.py

```diff
@@ -0,0 +1,36 @@
+import requests
+from bs4 import BeautifulSoup
+import json
+
+url = input("[+} URL : ")
+
+raw_code = requests.get(url).text
+
+# Beautiful : Translate text to html
+soup = BeautifulSoup(raw_code, "lxml")
+
+data_dictionary = dict()
+
+try:
+    website = soup.select(".biz-website > a")[0].string
+    data_dictionary["website"] = website
+except Exception as ex: 
+    data_dictionary["website"] = 'unknown'
+
+moreinfo_dictionary = dict()
+
+moreinfo = soup.select(".short-def-list > dl")
+for info in moreinfo:
+    attr_name, attr_content = "", ""
+    for  data in info.children:
+        if data.name == "dt": #tag_name == dt
```

Comment:
What do mean "dt" and "dd"?

Reply:
"dt" and "dd" is a kind of yelp html tag. Understand?

Reply:
Umm... I supposed "dt" means data type. right? I wonder if it's right and really don't know "dd" means.

Reply:
dd tag : https://www.w3schools.com/tags/tag_dd.asp
dt tag : https://www.w3schools.com/tags/tag_dt.asp

it is for your help.

Reply:
Oh, It's just a html tag. Thank you!